### PR TITLE
metering: Feature/add metering sink

### PR DESCRIPTION
**Pull request by cah-link**
this adds pushing events directly to the metering api

### CODE REVIEW

File: environments/kolla/files/overlays/ceilometer/event_pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+sources:
+    - name: event_source # 'source name'
+      events:
+          - "*" # 'event filter'
+      sinks:
+          - event_sink # 'sink name'
+sinks:
+    - name: event_sink # 'sink name'
+      publishers: # 'list of publishers'
```
Comment (artificial-intelligence) [inline environments/kolla/files/overlays/ceilometer/event_pipeline.yaml:10]: if we have inline comments, these should provide additional context.

these comments currently only replicate the yaml structure, e.g.

"sources -> name" becomes "source name" which does not yield any additional useful information, like what source is, or why does it need a name.

so in it's current form, these comments should imho all get removed.

**Review by artificial-intelligence — CHANGES_REQUESTED** [review]

File: environments/kolla/files/overlays/ceilometer/event_pipeline.yaml (hunk 1 above)
Comment (cah-link) [reply environments/kolla/files/overlays/ceilometer/event_pipeline.yaml]: Thanks for the hint. I put more context to these comments.

**Comment by berendt** [comment]
Failed DCO is fine. Will change this with a squash merge.

File: environments/kolla/files/overlays/ceilometer/event_pipeline.yaml (hunk 1 above)
Comment (cah-link) [reply environments/kolla/files/overlays/ceilometer/event_pipeline.yaml]: @artificial-intelligence Do you think this request is resolved?

**Comment by berendt** [comment]
Just merged the required backport: https://github.com/osism/container-image-kolla-ansible/commit/c7fa30374a68a1390a46de6741dd0ae2d248da90

**Review by artificial-intelligence — APPROVED** [review]
LGTM

**Comment by berendt** [comment]
Depends on https://review.opendev.org/c/openstack/kolla-ansible/+/894182

**Pull request merged**